rollup/Makefile

```diff
@@ -7,9 +7,22 @@ mock_abi:
 	cd .. && solc --evm-version cancun --bin --abi --optimize --overwrite -o ./build/bin ./rollup/mock_bridge/MockBridge.sol
 	cd .. && go run github.com/scroll-tech/go-ethereum/cmd/abigen --bin=./build/bin/MockBridge.bin --abi=./build/bin/MockBridge.abi --pkg=mock_bridge --out=./rollup/mock_bridge/MockBridge.go
 
-rollup_bins: ## Builds the Rollup bins.
+libzstd:
+	sudo mkdir -p /scroll/lib/
```

Comment:
what if a user is not su?
can we used home or some relative path?

Reply:
got it. yeah. it's clearer to separate customized libs from default libs.